Tests/BackendIntegrationTests/StoreKitIntegrationTests.swift

```diff
@@ -213,6 +213,15 @@ class StoreKit1IntegrationTests: BaseBackendIntegrationTests {
         expect(eligibility) == .eligible
     }
 
+    func testNoIntroOfferIfProductHasNoIntro() async throws {
```

Comment:
Confirmed this test failed on SK2 only:
<img width="471" alt="Screenshot 2022-08-22 at 16 30 33" src="https://user-images.githubusercontent.com/685609/186037337-5a7e8b8a-d522-4610-a926-23d89b0859d8.png">